client/test/e2e/inventory/inventory_list.spec.js

```diff
@@ -0,0 +1,76 @@
+/* global element, by, inject, browser */
+'use strict';
+
+const uuid   = require('node-uuid');
+const chai   = require('chai');
+const expect = chai.expect;
+
+const FU = require('../shared/FormUtils');
+const helpers = require('../shared/helpers');
+const components = require('../shared/components');
+
+helpers.configure(chai);
+
+describe.only('Inventory List ::', () => {
```

Comment:
Could you include a test that describes the behavior of the module when adding an item fails?  For example, create an invalid inventory item, try to submit, correct it to make it valid, then submit. 

That way, we know that all the models, inputs, states, etc, behave appropriately.